maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/service/PassportAssessmentService.java

```diff
@@ -0,0 +1,61 @@
+package gov.uk.courtdata.assessment.service;
+
+import gov.uk.courtdata.assessment.impl.PassportAssessmentImpl;
+import gov.uk.courtdata.assessment.mapper.PassportAssessmentMapper;
+import gov.uk.courtdata.dto.PassportAssessmentDTO;
+import gov.uk.courtdata.entity.PassportAssessmentEntity;
+import gov.uk.courtdata.exception.MAATCourtDataException;
+import gov.uk.courtdata.model.assessment.CreatePassportAssessment;
+import gov.uk.courtdata.model.assessment.UpdatePassportAssessment;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class PassportAssessmentService {
+
+    private final PassportAssessmentImpl passportAssessmentImpl;
+    private final PassportAssessmentMapper passportAssessmentMapper;
+
+    public PassportAssessmentDTO find(Integer passportAssessmentId) {
+        PassportAssessmentEntity passportAssessmentEntity = passportAssessmentImpl.find(passportAssessmentId);
+        return buildPassportAssessmentDTO(passportAssessmentEntity);
+    }
+
+    @Transactional(rollbackFor = MAATCourtDataException.class)
+    public PassportAssessmentDTO update(UpdatePassportAssessment updatePassportAssessment) {
+        log.info("Update Passport Assessment - Transaction Processing - Start");
+        PassportAssessmentDTO passportAssessmentDTO =
+                passportAssessmentMapper.updatePassportAssessmentToPassportAssessmentDTO(updatePassportAssessment);
+        log.info("Updating existing passport assessment record");
+        PassportAssessmentEntity passportAssessmentEntity = passportAssessmentImpl.update(passportAssessmentDTO);
+        log.info("Update Passport Assessment - Transaction Processing - End");
+        return buildPassportAssessmentDTO(passportAssessmentEntity);
+    }
+
+    public void delete(Integer passportAssessmentId) {
+        passportAssessmentImpl.delete(passportAssessmentId);
+    }
+
+    @Transactional(rollbackFor = MAATCourtDataException.class)
+    public PassportAssessmentDTO create(CreatePassportAssessment createPassportAssessment) {
+        log.info("Create Passport Assessment - Transaction Processing - Start");
+        PassportAssessmentDTO passportAssessmentDTO =
+                passportAssessmentMapper.createPassportAssessmentToPassportAssessmentDTO(createPassportAssessment);
+        log.info("Creating new passport assessment record");
```

Comment:
Before creating a new assessment, we should first check for existing incomplete passport/hardship assessments for the rep order and raise an error if any exist.

Reply:
I think these would go to means assessment service as it is business logic, as part of validations